### ISSUE TITLE
Does it work on Windows 10?

### ISSUE THREAD

**Issue by zeroamps** [Alex079/vscode-avr-helper#8]
Hi Alex,

Does you plugin work on Windows 10?
I've installed MinGW so I have sh, make utilities and AVR toolchain installed with Arduino.
I just want to build a very easy code.

int main(void)
{
    return 0;
}

and it throws this error.

/bin/sh: -c: line 0: syntax error near unexpected token `('
/bin/sh: -c: line 0: `C:\Program Files (x86)\Arduino\hardware\tools\avr\bin\avr-gcc.exe -DF_CPU=16000000UL -mmcu=atmega328p -pipe  -MM main.c'
/bin/sh: -c: line 0: syntax error near unexpected token `('
/bin/sh: -c: line 0: `C:\Program Files (x86)\Arduino\hardware\tools\avr\bin\avr-gcc.exe -DF_CPU=16000000UL -mmcu=atmega328p -pipe  -MM main.c'
===== Making .vscode/avr.build/obj/main.o

**Comment by chovanj** [comment]
Hi Alex, 

I played with that a bit and it looks that the variable AVR.source.compiler doesn't know to process Windows paths.
In VS code configuration file it's like this "compilerPath": "C:\\Program Files (x86)\\Arduino\\hardware\\tools\\avr\\bin\\avr-gcc.exe".
But your configuration file avr.properties.mk doesn't like this AVR.source.compiler = "C:\\Program Files (x86)\\Arduino\\hardware\\tools\\avr\\bin\\avr-gcc.exe"

So the only solution I found was to add the path to the System Variables and call just avr-gcc.exe without path that works.

**Comment by Alex079** [comment]
Hi Jozef,

When working on the extension, I faced a problem that "make" did not process whitespace correctly inside variable values.
See https://stackoverflow.com/questions/9838384/can-gnu-make-handle-filenames-with-spaces for example.
Whitespace is always treated as list item separator and screening it didn't help, unfortunately.

It looks like your solution should work as long as other avr tools are on the system path. Thank you for reporting the problem. It impacts not only Windows.

I created #7 to change to a more flexible build tool instead of Makefile, possibly Jakefile. It looks like VSCode now includes Jakefile tasks support out of the box.
